qiskit/utils/deprecation.py

```diff
@@ -27,7 +27,7 @@ def deprecate_func(
     additional_msg: str | None = None,
     pending: bool = False,
     package_name: str = "qiskit",
-    removal_timeline: str = "no earlier than 3 months after the release date",
+    removal_timeline: str = "in the next major release",
```

Comment:
This works fine, but we should have sufficient info to compute the next major version from the `since` field. We could set this to `None` and then do something like:

```
if removal_timeline is None:
    removal_major = int(since.split(",")[0]) + 1
    removal_timeline = f"in the next major release {removal_major}"
```

Or if you're worried about deprecated features passing a major version boundary (e.g. we deprecate something in 1.0 and don't remove it until 3.0) we can replace since with the package's `__version__`.

Reply:
good one!